ignite/services/chain/init.go

```diff
@@ -15,20 +15,45 @@ import (
 	"github.com/ignite/cli/ignite/pkg/events"
 )
 
+type (
+	// InitArgs represents argument to add additional initialization for the chain
+	// InitAccounts initializes chain accounts from the Ignite config
+	// InitConfiguration initializes node configuration from the Ignite config
+	// InitGenesis initializes genesis state for the chain from Ignite config
+	InitArgs struct {
+		InitAccounts      bool
+		InitConfiguration bool
+		InitGenesis       bool
+	}
+)
+
 const (
 	moniker = "mynode"
 )
 
+var (
+	// InitArgsAll performs all initialization for the chain
+	InitArgsAll = InitArgs{
+		true,
+		true,
+		true,
+	}
+
+	// InitArgsNone performs minimal initialization for the chain by only initializating a node
+	InitArgsNone = InitArgs{
+		false,
+		false,
+		false,
+	}
+)
```

Comment:
I never used this type of combined arguments before so I'm figuring out if it's worth it of it would be a good practice.

The following is just an opinion on the struct type for arguments "pattern".

Here I see that you're trying to give some semantics to it, which is really good, but the downside is that in this case the variadic options might be better suited; Defining argument combinations like this starts to seem a bit like defining options for the variadic pattern.

Another opinion, when using this pattern I would just use a single struct, `InitArgs`, to keep it simple, which should be the benefit over the variadic options I believe, but then we would potentially end up with cases that loose clear semantics like:

```go
c.Init(ctx, InitArgs{true, true, true})
```

A solution could be to define a variable or maybe better inline like:

```go
c.Init(ctx, InitArgs{
	InitAccounts: true,
	InitConfigurations: true,
	InitGenesis: true,
})
```

Personally I would lean towards inlining.